_includes/_shared_global_header.html

```diff
@@ -37,13 +37,13 @@
 <li><a href="/getstarted/" data-automation-id="sh-getstarted">Get Started</a></li>
 <li><a href="/groups/" data-automation-id="sh-groups">Groups</a></li>
 <li><a href="/serve/" data-automation-id="sh-servingopportunities">Serving Opportunities</a></li>
-<li><a href="http://www.crossroads.net/reachout/go" data-automation-id="sh-go">GO Somewhere</a></li>
+<li><a href="http://int.crossroads.net/reachout/go" data-automation-id="sh-go">GO Somewhere</a></li>
```

Comment:
Even though the shared header isn't using this code, should we point these at prod again?